fabric_plugin/tests/blueprint/blueprint.yaml

```diff
@@ -45,7 +45,21 @@ node_templates:
                         process:         { get_input: process }
                         fabric_env:      { get_input: fabric_env }
                         custom_env_var:  { get_input: custom_input }
-
+            test_sudo:
```

Comment:
rather than add an interface, why not simply add in input `use_sudo` and pass it where you need it?